client/src/Command/ProcessLayCSVCommand.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Command;
+
+use App\Service\Client\RestClient;
+use App\Service\CsvUploader;
+use App\Service\DataImporter\CsvToArray;
+use App\Service\File\Storage\S3Storage;
+use App\Service\Mailer\Mailer;
+use Aws\S3\Exception\S3Exception;
+use Aws\S3\S3Client;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
+use Throwable;
+
+class ProcessLayCSVCommand extends Command {
+    protected static $defaultName = 'digideps:process-lay-csv';
+
+    private const CHUNK_SIZE = 50;
+
+    private array $output = [
+        'errors' => [],
+        'added' => 0,
+        'skipped' => 0,
+    ];
+
+    public function __construct(
+        private S3Client $s3,
+        private RestClient $restClient,
+        private ParameterBagInterface $params,
+        private Mailer $mailer,
+        private LoggerInterface $logger,
+    ) {
+        parent::__construct();
+    }
+
+    protected function configure(): void {
+        $this
+            ->setDescription('Process the Lay Deputies CSV from the S3 bucket')
+            ->addArgument('email', InputArgument::REQUIRED, 'Email address to send results to');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $bucket = $this->params->get('s3_sirius_bucket');
+        $layDeputyReportFile = 'layDeputyReport.csv';
+
+        try {
+            $this->s3->getObject([
+                'Bucket' => $bucket,
+                'Key' => $layDeputyReportFile,
+                'SaveAs' => "/tmp/layReport.csv"
```

Comment:
Make sure you add code to delete the csv from the container once the process finishes or errors, otherwise personal data might get left around on the file system.